storagetransfer/azure_request.py

```diff
@@ -0,0 +1,126 @@
+    Storage."""
+
+    client = storage_transfer.StorageTransferServiceClient()
+
+    # The ID of the Google Cloud Platform Project that owns the job
+    # project_id = 'my-project-id'
+
+    # A useful description for your transfer job
+    # description = 'My transfer job'
+
+    # Azure Storage Account name
+    # azure_storage_account = 'accountname'
+
+    # Azure Shared Access Signature token
+    # azure_sas_token = '?sv=...'
+
+    # Azure Blob source container name
+    # source_container = 'my-azure-source-bucket'
+
+    # Google Cloud Storage destination bucket name
+    # sink_bucket = 'my-gcs-destination-bucket'
+
+    now = datetime.utcnow()
+    # Setting the start date and the end date as
+    # the same time creates a one-time transfer
+    one_time_schedule = {
+        'day': now.day,
+        'month': now.month,
+        'year': now.year
+    }
+
+    transfer_job_request = storage_transfer.CreateTransferJobRequest({
+        'transfer_job': {
+            'project_id': project_id,
+            'description': description,
+            'status': storage_transfer.TransferJob.Status.ENABLED,
+            'schedule': {
+                'schedule_start_date': one_time_schedule,
+                'schedule_end_date': one_time_schedule
+            },
+            'transfer_spec': {
+                'azure_blob_storage_data_source': {
+                    'storage_account': azure_storage_account,
+                    'azure_credentials': {
+                        'sas_token': azure_sas_token
+                    },
+                    'container': source_container
+                },
+                'gcs_data_sink': {
+                    'bucket_name': sink_bucket,
+                }
+            }
+        }
+    })
+
+    result = client.create_transfer_job(transfer_job_request)
+    print(f'Created transferJob: {result.name}')
+
+
+# [END storagetransfer_transfer_from_azure]
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument(
+        '--project-id',
+        help='The ID of the Google Cloud Platform Project that owns the job',
+        required=True)
+    parser.add_argument(
+        '--description',
+        help='A useful description for your transfer job',
+        default='My transfer job')
+    parser.add_argument(
+        '--azure-storage-account',
+        help='Azure Storage Account name',
+        required=True)
+    parser.add_argument(
+        '--azure-sas-account',
+        help='Azure Shared Access Signature token',
+        required=True)
+    parser.add_argument(
+        '--source-container',
+        help='Azure Blob source container name',
+        required=True)
+    parser.add_argument(
+        '--sink-bucket',
+        help='Google Cloud Storage destination bucket name',
+        required=True)
+
+    args = parser.parse_args()
+
+    create_one_time_azure_transfer(**vars(args))
```

Comment:
New samples should not include CLIs: https://googlecloudplatform.github.io/samples-style-guide/#no-cli